src/main/java/com/khutircraftubackend/auth/FinalMessagesUsers.java

```diff
@@ -0,0 +1,16 @@
+package com.khutircraftubackend.auth;
+
+public final class FinalMessagesUsers {
```

Comment:
from English point if view the better name for this class could be:  `AuthReposnseMessages`